### ISSUE TITLE
Plans to support latest version of VS for Mac

### ISSUE THREAD

**Issue by ChasakisD** [jsuarezruiz/VS4Mac-SkiaSharpFiddle#3]
Hi, 

Are there plans to support latest version of VS for Mac? This tool is very helpful when u want to have a right preview of skia. Official skia fiddle is in C++ so having a C# editor right in your visual studio is pretty useful.

**Comment by jsuarezruiz** [comment]
Yes, of course. I will update the addin to support the latest versions very soon.

**Comment by ChasakisD** [comment]
@jsuarezruiz any news about that?
